backend/cabin-sorting/hackerSortingAlgo.ts

```diff
@@ -30,6 +31,13 @@ function loadCSV (filepath: string, headers: boolean): any[] {
   return parse(fileContent, options)
 }
 
+async function loadFromDatabase (): Promise<any[]> {
+    const response = await axios.get("http://localhost:4000/sortedHackers")
```

Comment:
can we put the URL in an .env variable instead so it isn't hardcoded to localhost:4000